disco/bot/bot.py

```diff
@@ -46,6 +46,12 @@ class BotConfig(Config):
     commands_prefix : str
         A string prefix that is required for a message to be considered for
         command parsing.
+    commands_prefixes : list[string]
```

Comment:
If we're adding this configuration option, we need to deprecate `commands_prefix`. This probably means we need to throw a DeprecationWarning when we read/validate the config inside of the bot constructor.